libraries/stdlib/jvm/build.gradle

```diff
@@ -79,6 +79,8 @@ dependencies {
     testApi project(':kotlin-test:kotlin-test-junit')
 
     builtins project(':core:builtins')
+
+    testImplementation("com.google.code.gson:gson:2.8.9")
```

Comment:
In order to have the same version of dependency across all the modules, it is better to declare it this way:
```
testImplementation(commonDependency("com.google.code.gson:gson"))
```

Reply:
Thanks for the comment! Unfortunately `libraries/stdlib/jvm/build.gradle` is a groovy script and you can't use the `commonDependency(...)` function there (as it is defined in `DependenciesKt#commonDependency` and probably can't be directly imported). What would be the best way to approach this issue?